sdk/servicebus/service-bus/samples/README.md

```diff
@@ -1,47 +1,25 @@
-# Getting started with samples #
+# Getting started with samples
 
 ## Install the library
 
-There are 2 ways you can work with the samples.
-- Copy the sample file you want to a folder of your choice and run `npm install @azure/service-bus`
-to install the library
-- Or, in case you have cloned this repo, then run `npm install` in the `samples` folder to install the
-library. This will install the library as well as other dependencies that are required by some of the
-samples.
+There are three ways to work with the samples.
+- download the sample package from [docs.microsoft.com/samples](https://docs.microsoft.com/samples) by browsing to the Azure Service Bus product and choosing either JavaScript or TypeScript in the list of languages
+- copy the sample file you want to a folder of your choice and use `npm` to install its requisite dependencies, then edit the file to add the necessary credentials and information, finally running it with NodeJS
+- in case you have cloned this repo, run `npm install` in either the `samples/javascript` or `samples/typescript` folder to install the library as well as other dependencies that are required by some of the samples
```

Comment:
```suggestion
- In case you have cloned this repo, run `npm install` in either the `samples/javascript` or `samples/typescript` folder to install the library as well as other dependencies that are required by some of the samples.
```